test/services/journeyAnswers/NICsAnswersServiceImplSpec.scala

```diff
@@ -122,13 +122,13 @@ class NICsAnswersServiceImplSpec extends TableDrivenPropertyChecks with AnyWordS
 
           val expectedResult = buildExpectedRequestResult(expectedApiData)
 
-          val result = service.saveClass4SingleBusiness(journeyCtxWithNino, class4SingleBusinessAnswers).value.futureValue
+          val result = service.saveClass4BusinessData(journeyCtxWithNino, answer).value.futureValue
 
           assert(result.isRight)
           assert(connector.upsertAnnualSummariesSubmissionData === expectedResult)
         }
       }
-      "overriding existing details" in new StubbedService {
+      s"overriding existing details for $testDescription" in new StubbedService {
```

Comment:
undo this line and move it inside the braces, same as the `"creating a new answers model when there is no existing Class 4 data" in new StubbedService {` test